sunpy/time/astropy_time.py

```diff
@@ -93,3 +94,10 @@ def strftime(self, format_spec):
                 return formatted_strings[0]
             else:
                 return np.array(formatted_strings).reshape(self.shape)
+
+
+def is_time_equal(t1, t2):
```

Comment:
is it a good idea to make this part of the public API, is it not just a work around for the comparison bug in astropy?

Reply:
I'll add a `_`. :+1:

Reply:
also remove it from the `__all__`

Reply:
Then how will I import it?